pyproject.toml

```diff
@@ -0,0 +1,195 @@
+  {name = "Tyler Baker" , email = "tyler@opensourcefoundries.com"},
+  {name = "William K Lin" , email = "lin.wen.kuang@gmail.com"},
+  {name = "alan bennett" , email = "akbennett@hasdell"},
+  {name = "isaiahduncan" , email = "32970507+isaiahduncan@users.noreply.github.com"},
+  {name = "lollivier" , email = "lollivier@baylibre.com"},
+  {name = "Alexandra" , email = "alexandra@arch.localhost"},
+  {name = "Alexandre Belloni" , email = "alexandre.belloni@free-electrons.com"},
+  {name = "Alice Ferrazzi" , email = "alice.ferrazzi@cybertrust.co.jp"},
+  {name = "Ana Guerrero Lopez" , email = "ana@ekaia.org"},
+  {name = "AngeloGioacchino Del Regno" , email = "kholk11@gmail.com"},
+  {name = "Ard Biesheuvel" , email = "ard.biesheuvel@linaro.org"},
+  {name = "Bastian Krause" , email = "bst@pengutronix.de"},
+  {name = "Bjorn Andersson" , email = "bjorn.andersson@linaro.org"},
+  {name = "Denys Fedoryshchenko" , email = "nuclearcat@nuclearcat.com"},
+  {name = "Fathi Boudra" , email = "fathi.boudra@linaro.org"},
+  {name = "Gustavo Padovan" , email = "gustavo.padovan@collabora.com"},
+  {name = "Heidi Fahim" , email = "heidifahim@google.com"},
+  {name = "Jakob Unterwurzacher" , email = "jakob.unterwurzacher@theobroma-systems.com"},
+  {name = "Jan-Simon Möller" , email = "dl9pf@gmx.de"},
+  {name = "Kees Cook" , email = "keescook@chromium.org"},
+  {name = "Lakshmipathi.G" , email = "Lakshmipathi.Ganapathi@Collabora.com"},
+  {name = "Matt Hart" , email = "matthew.hart@arm.com"},
+  {name = "Matthias Maennich" , email = "maennich@google.com"},
+  {name = "Maxime Ripard" , email = "maxime.ripard@free-electrons.com"},
+  {name = "Mylène Josserand" , email = "mylene.josserand@collabora.com"},
+  {name = "Neil Williams" , email = "codehelp@debian.org"},
+  {name = "Nick Desaulniers" , email = "nickdesaulniers@users.noreply.github.com"},
+  {name = "Nikolai Kondrashov" , email = "Nikolai.Kondrashov@redhat.com"},
+  {name = "Nikolay Yurin" , email = "yurinnick@meta.com"},
+  {name = "Quentin Schulz" , email = "q.schulz@gmail.com"},
+  {name = "Ricardo Ribalda" , email = "ricardo@ribalda.com"},
+  {name = "Riku Voipio" , email = "riku.voipio@linaro.org"},
+  {name = "Robin Lu" , email = "robin.lubin@shopee.com"},
+  {name = "Sasha Levin" , email = "sashal@kernel.org"},
+  {name = "Shreeya Patel" , email = "shreeya.patel@collabora.com"},
+  {name = "Sjoerd Simons" , email = "sjoerd.simons@collabora.co.uk"},
+  {name = "Thomas Petazzoni" , email = "thomas.petazzoni@free-electrons.com"},
+  {name = "Vince Hillier" , email = "vince@revenni.com"},
+  {name = "Wang Mingyu" , email = "wangmy@cn.fujitsu.com"},
+  {name = "nkrishna" , email = "nkrishna@otc-prt.org"},
+  {name = "roxell" , email = "anders.roxell@gmail.com"},
+  {name = "sjoerd-ccu" , email = "sjoerd.simons@collabora.co.uk"}
+]
+maintainers = [
+  {name = "Denys Fedoryshchenko", email = "denys.f@collabora.com"},
+  {name = "Jeny Sadadia", email = "jeny.sadadia@collabora.com"}
+]
```

Comment:
Arisu is listed as co-maintainer too: https://kernelci.org/docs/org/maintainers/#core-tools